### PR TITLE
[NP-2234] - Text Overlap Fix

### PR DESCRIPTION
**Pull request by nanopay-arthur**
Ref: https://nanopay.atlassian.net/browse/NP-2234
#TimeSpent: 4h

Screen Shot 2020-10-22 at 5 42 01 PM
Screen Shot 2020-10-22 at 5 42 09 PM
Screen Shot 2020-10-22 at 5 40 29 PM

### CODE REVIEW

**Review by kennyqiyenkan — APPROVED** [review]

File: src/foam/u2/detail/SectionedDetailPropertyView.js [hunk 1]
```diff
@@ -228,13 +230,22 @@ foam.CLASS({
         .addClass(`sectioned-detail-property-${this.prop.name}`)
         .add(this.slot(function(mode, prop, prop$label) {
 
+          var isCheckBox = false;
+          if ( typeof prop.view === "function" ) {
```
Comment (kgrgreer) [inline src/foam/u2/detail/SectionedDetailPropertyView.js]: foam.Function.isInstance() is more consistent.

**Comment by Nauna** [comment]
I wonder is there anyway to make the validation section size itself based on the size of the top container. I think adding checks for Checkbox explicitly can be troublesome.

**Comment by nanopay-arthur** [comment]
> I wonder is there anyway to make the validation section size itself based on the size of the top container. I think adding checks for Checkbox explicitly can be troublesome.

@Nauna So the issue wasn't really the validation section but rather the text section of the checkbox. The height of the top container wasn't being updated correctly due to the label text being position: absolute. I wasn't able to come up with another fix for this, but I agree, doing an explicit check for checkbox is not pretty.

**Comment by kgrgreer** [comment]
Has merge conflict. Please resolve.

**Pull request merged**